galpy/orbit/RZOrbit.py

```diff
@@ -480,7 +480,7 @@ def _integrateRZOrbit(vxvv,pot,t,method,dt):
                 warnings.warn("Cannot use C integration because C extension not loaded (using %s instead)" % (method), galpyWarning)
             else:
                 warnings.warn("Cannot use C integration because some of the potentials are not implemented in C (using %s instead)" % (method), galpyWarning)
-    if method.lower() == 'leapfrog' \
+    if method.lower() == 'leapfrog' or method.lower() == 'dop853' \
```

Comment:
Should move the ``dop853`` case to the elif case for ``odeint`` below (with Dormand-Prince, we can use the ``_RZEOM`` equation of motion)